packages/platform-core/src/errors/amplify_error.ts

```diff
@@ -191,6 +182,9 @@ export abstract class AmplifyError<T extends string = string> extends Error {
         error
       );
     }
+    if (error instanceof Error && isAmplifyUserError(error)) {
```

Comment:
```suggestion
    if (AmplifyError.isAmplifyError(error) && error.classification === 'ERROR') {
```

or just `if (AmplifyError.isAmplifyError(error)) {` in case this also re-maps known faults?

and . should this be very first line in `fromError` ?

Reply:
yes, updated.